swedish-oidc-profile.md

```diff
@@ -212,8 +209,66 @@ An OpenID Provider compliant with this profile SHOULD support the PKCE extension
 
 An OpenID Provider MUST NOT allow a Relying Party to use the `plain` code challenge method.
 
+<a name="authentication-request-parameter-extensions"></a>
+### 2.2. Authentication Request Parameter Extensions
+
+This section contains authentication request parameter extensions defined within the scope of this profile.
+
+<a name="client-provided-user-message"></a>
+#### 2.3.1. Client Provided User Message
+
+**Parameter:** `https://id.oidc.se/param/userMessage`
```

Comment:
Need to ask - will we drop 1.0 from all identifiers? I.e - will we have a naming convention with the new common domain `id.oidc.se` and then use `param`, `claim`, `scope` etc in singular?

Example, old => new:
```
https://claims.oidc.se/1.0/userMessage => https://id.oidc.se/param/userMessage
https://claims.oidc.se/1.0/personalNumber => https://id.oidc.se/claim/personalNumber
https://scopes.oidc.se/
1.0/naturalPersonName => https://id.oidc.se/scope/naturalPersonName
```

Just trying to get a grasp on the overall naming convention.